README.md

```diff
@@ -33,3 +33,109 @@ Prepare the package:
 
     
+# Docker
+
+## Build
+
+### Local git version
+
+```
+docker build --build-arg version_reg_app=$(git rev-parse HEAD) \
+             --file Dockerfile.local \
+             --tag pds_registry_app:$(git rev-parse HEAD) \
+             .
+```
+
+## Run
+
+### Elastic Search
+
+1. prepare `mkdir /tmp/es /tmp/output`  
+    These two directories are to store the data between steps. If there is an existng elastic search database or harvest working area, then there is no need for this step.
+1. fetch `docker pull elasticsearch:7.10.1`  
+    Only need to fetch the image once.
+1. run  
+    ```
+    docker run --detach \
+               --env "discovery.type=single-node" \
+               --name es \
+               --network pds \
+               --publish 9200:9200 \
+               --publish 9300:9300 \
+               --rm \
+               --user $UID \
+               --volume /tmp/es:/usr/share/elasticsearch/data \
+               elasticsearch:7.10.1
+    ```  
+    Start the elastic search engine and make sure that any of the data created is owned by yourself so that you can clean it up. If you are not using the /tmp/es directory created above, then make sure to change the --volume argument as needed so that elastic search has access to it. Also change the --user arguement such that elastic search has full privledges to read, write, and modify the data.
+
+### Install test data 
+
+This section will use the small test data set contained within the docker image. The steps used to do this can be used as a template to install your own more complicated datasets as well.
+
+First, install the registry schema for elastic search. Things to note, the URL host name is the --name argument value used when starting elastic search.
+
+```
+docker run --network pds \
+           --rm \
+           --user $UID \
+           pds_registry_app:$(git rev-parse HEAD) \
+              registry-manager create-registry -es http://es:9200 \
+                                               -schema /var/local/registry/elastic/registry.json 
+```
+
+Second we need to harvest the test data. It is currently setup in /var/local/harvest/archive. If one wanted to use their own archive of data, they would simply have to add `--volume /path/to/archive:/var/local/harvest/archive` to the docker run portion of this command. Because we cannot easily run two commands in one docker run invokation, it is necessary to use an external directory for /var/local/havest/output. The ephemeral nature of a container erases what we would write into /var/local/harvest/output between the two docker run invokations.
+```
+docker run --network pds \
+           --rm \
+           --user $UID \
+           --volume /tmp/output:/var/local/harvest/output \
+           pds_registry_app:$(git rev-parse HEAD) harvest \
+              -c /var/local/harvest/conf/examples/registry.xml \
```

Comment:
I think this means that you updated this example configuration file for the whole repository. That is fine as it works this way. For a more sustainable configuration I would think of a configuration file being update as part as the Dockerfile.local to match the deployment configuration there.